azure-pipelines.yml

```diff
@@ -696,38 +712,53 @@ jobs:
 - job: TestRpmPackage
   displayName: Test Rpm Package
   timeoutInMinutes: 120
-  dependsOn: BuildRpmPackageCentOS7
+  dependsOn: BuildRpmPackages
   condition: and(succeeded(), in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'Manual', 'Schedule'))
   pool:
     vmImage: 'ubuntu-20.04'
+  strategy:
+    matrix:
+      CentOS7:
+        image: centos
+        tag: centos7
+        distro: el7
+        artifact: yum
+      Fedora35:
+        image: fedora
+        tag: 35
+        distro: fc35
+        artifact: rpm-fedora35
```

Comment:
No need to define another variable `artifact` as it can be constructed from `image` and `tag`.

Reply:
CentOS7 uses different pattern, so I have to write it explicitly.

Reply:
CentOS 7 has been dropped.